experiments/AtmosLES/convective_bl_model.jl

```diff
@@ -279,20 +304,26 @@ function convective_bl_model(
                 turbconv = turbconv_bcs(turbconv)[1],
             ),
             AtmosBC(turbconv = turbconv_bcs(turbconv)[2]),
-        ),
+        )
+    end
+
+    moisture_flux = FT(0)
+    problem = AtmosProblem(
         init_state_prognostic = ics,
+        boundaryconditions = boundary_conditions,
     )
 
     # Assemble model components
     model = AtmosModel{FT}(
-        AtmosLESConfigType,
+        config_type,
         param_set;
         problem = problem,
         turbulence = SmagorinskyLilly{FT}(C_smag),
-        moisture = EquilMoist{FT}(; maxiter = 5, tolerance = FT(0.1)),
-        source = source,
+        moisture = moisture,
+        source = source_default,
```

Comment:
```suggestion
        source = source,
```
It looks like we want `source` and not `source_default`

Reply:
Does that me and that in 236 it should be source rather than source_default as well?